### PR TITLE
Update to deploy to LinkedIn articatory

### PR DESCRIPTION
**Pull request by kkoser**
This updates the build script to deploy to the LinkedIn artifactory instance,
instead of going straight to jcenter.

### CODE REVIEW

**Comment by ethankhall** [comment]
lgtm

**Comment by kkoser** [comment]
I need to update RELEASING.md before this is merged.

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 GROUP_ID=com.linkedin.shaky
 ARTIFACT=shaky
-VERSION_NAME=1.0.2-SNAPSHOT
```
Comment (drewhannay) [inline gradle.properties]: Shouldn't this still have SNAPSHOT for local development? Otherwise the steps in RELEASING.md don't make sense.

Comment (kkoser) [inline gradle.properties]: Good catch, fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,3 +3,8 @@
 ## Version 1.0.0
 
 Initial release.
+
+## Version 1.0.2
+
+Fixed small image bug
```
Comment (drewhannay) [inline CHANGELOG.md]: Include the github issue number

File: build.gradle [hunk 3]
```diff
@@ -29,3 +30,5 @@ ext {
 
     isCi = "true".equals(System.getenv('CI'))
```
Comment (drewhannay) [inline build.gradle]: Is this still needed?

File: gradle.properties (hunk 1 above)
Comment (drewhannay) [inline gradle.properties]: Not fixed anymore :)

File: buildSrc/src/main/groovy/com/linkedin/gradle/DistributeTask.groovy [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        def buildName = convention.clientConfig.info.buildName
+        def context = convention.clientConfig.publisher.contextUrl
+        def password = convention.clientConfig.publisher.password
+
+        def body = [
+                "publish"              : "true",
+                "overrideExistingFiles": "false",
+                "async"                : "true",
+                "targetRepo"           : "maven",
+                "sourceRepos"          : ["pygradle"],
```
Comment (ethankhall) [inline buildSrc/src/main/groovy/com/linkedin/gradle/DistributeTask.groovy]: This should probably be `shaky-android`

**Pull request merged**